client/src/components/core/ms-tooltip/index.ts

```diff
@@ -0,0 +1,7 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+export * from '@/components/core/ms-tooltip/utils';
+
+import MsInformationTooltipIcon from '@/components/core/ms-tooltip/MsInformationTooltipIcon.vue';
+
+export { MsInformationTooltipIcon };
```

Comment:
maybe we could name it only MsInformationTooltip?
The same way MsDropdown is a button that calls MsDropdownPopover, here the MsInformationTooltip would represent the global UI component even if technically it's one component calling a second one.